encryption-service-jasypt/README.md

```diff
@@ -6,27 +6,24 @@ Vault encryption service implementation
 
 ```yaml
 encryptionService:
-  class: "com.mx.path.facility.security.vault.VaultEncryptionService"
+  class: com.mx.path.service.facility.encryption.jasypt.JasyptEncryptionService
```

Comment:
I think this should be `com.mx.path.service.facility.encryption.JasyptEncryptionService`, maybe?